spec/std/env_spec.cr

```diff
@@ -172,4 +173,85 @@ describe "ENV" do
   ensure
     ENV.delete("FOO")
   end
+
+  it "clears the env" do
```

Comment:
```suggestion
  it ".clear" do
```

Reply:
changed to `it "does .clear" do` to be consistent with other several other specs in this file